skrifa/src/meta/metrics.rs

```diff
@@ -0,0 +1,434 @@
+//! Global font and glyph specific metrics.
+
+use read_fonts::{
+    tables::{glyf::Glyf, hmtx::LongMetric, hvar::Hvar, loca::Loca},
+    types::{BigEndian, GlyphId},
+    TableProvider,
+};
+
+use crate::NormalizedCoord;
+
+/// Metrics for a text decoration.
+#[derive(Copy, Clone, PartialEq, Default, Debug)]
+pub struct Decoration {
+    /// Offset to the top of the decoration from the baseline.
+    pub offset: f32,
+    /// Thickness of the decoration.
+    pub thickness: f32,
+}
+
+/// Extents of a rectangular region.
+#[derive(Copy, Clone, PartialEq, Default, Debug)]
+pub struct BoundingBox {
+    /// Minimum x-coordinate.
+    pub x_min: f32,
+    /// Minimum y-coordinate.
+    pub y_min: f32,
+    /// Maximum x-coordinate.
+    pub x_max: f32,
+    /// Maximum y-coordinate.
+    pub y_max: f32,
+}
+
+/// Metrics that apply to all glyphs in a font.
```

Comment:
Suggest clarifying if this is for the entire design space or for a specific position in design space?

Reply:
Sugest clarifying with links that this data is drawn from https://learn.microsoft.com/en-us/typography/opentype/spec/head, etc.

Reply:
It’s for a specific position in design space. I’ll add a note about that and the relevant links.

Reply:
Done!